CredentialManager/app/src/main/java/com/google/credentialmanager/sample/Auth.kt

```diff
@@ -108,6 +120,13 @@ class Auth(context: Context) {
         activity: Activity,
         requestResult: JSONObject
     ): CreatePublicKeyCredentialResponse? {
+
+        if(!isPasskeySupported(activity)) {
+            Toast.makeText(activity, "Passkeys are not supported on this devices", Toast.LENGTH_SHORT)
```

Comment:
Make last word singular "device"

Reply:
Would it be better to make these a reference to strings.xml?

Reply:
Updatd